options/bsd/include/mlibc/bsd-sysdeps.hpp

```diff
@@ -5,6 +5,8 @@ namespace [[gnu::visibility("hidden")]] mlibc {
 
 [[gnu::weak]] int sys_brk(void **out);
 
+[[gnu::weak]] int sys_getloadavg(int *nsample_out, double *samples, int nsample);
```

Comment:
I'd prefer to force the sysdep to always return 3 samples on success. `getloadavg()` (not the sysdep) can handle the case that the user passes n < 3.